website/content/blog/2025-08-28-openbao-oss-europe-2025-wrap.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "It's a wrap! OpenBao at Open Source Summit Europe 2025"
+description: "OpenBao joined the Open Source Summit EU 2025 in Amsterdam and gave roadmap previews, Q&A, and hands-on conversations about open-source secrets management."
+slug: wrap-openbao-oss-europe-2025
+authors: ChristophVoigt
+tags: [announcement, conferences, community]
+image: /img/openbao-osseu-2025.jpg
+---
+
+The OpenBao community had a great time at **[Open Source Summit Europe 2025](https://events.linuxfoundation.org/open-source-summit-europe/)**! The event once again showed how strong the demand has become for secure and reliable secret management in enterprise environments.
+
+One theme was clear: companies and individuals are in demand to manage secrets across their systems, and solutions like OpenBao are foundational for security and compliance.
+
+![OpenBao Maintainers](/img/openbao-osseu-2025.jpg)
+
+Throughout the conference, we shared how OpenBao is able to help organizations to run production workloads securely. We showcased existing features such as strong encryption, fine-grained access controls and namespace isolation. We also gave a glimpse into what’s next: declarative initialization, improvements around namespaces and extended key management capabilties.
+
+Furthermore, the conference was also a wonderful opportunity for OpenBao maintainers to meet in person, exchange ideas, and align on next steps. A big thank you goes to [**Adfinis**](https://adfinis.com/), [**Paymenttools**](https://www.paymenttools.com/) and [**Reply**](https://www.reply.com/), who supported the project by sending maintainers to represent OpenBao on-site.
+
+And finally, a heartfelt thanks to everyone who stopped by, joined discussions, or offered feedback. Your energy and curiosity keep the project moving forward. To everyone who was not able to join the conference, feel free to learn more at [**openbao.org**](https://openbao.org/) or join the conversation on [**Github**](https://github.com/openbao).
```

Comment:
Should we mention and/or directly link to the contribution guide which also helps people finding the Matrix chat and MLs?

* https://openbao.org/docs/contributing/

Nit: `s/Github/GitHub/`